explorer/src/lib/components/transaction-details/TransactionDetails.css

```diff
@@ -38,7 +61,12 @@
     grid-template-columns: minmax(0, auto) minmax(0, 3fr);
   }
 
-  .transaction-details__payload {
+  .transaction-details__memo {
+    margin-bottom: 1rem;
```

Comment:
Why is that?
<img width="947" alt="Screenshot 2024-10-02 at 13 07 36" src="https://github.com/user-attachments/assets/0bbc99b2-7a6c-45dd-b55e-4a2b9f805d18">

It looks better to me if aligned?

<img width="943" alt="Screenshot 2024-10-02 at 13 07 46" src="https://github.com/user-attachments/assets/c69bc838-6498-4007-832a-0559fd78c643">

Reply:
Did not notice the misalignment there, the margin is needed to create space between the decoded memo and opened payload.